pages/api/updateAlgolia.ts

```diff
@@ -0,0 +1,15 @@
+import path from 'path';
+import fs from 'fs';
+import url from 'url';
+import { updateIndex } from '@/lib/algolia/getRecords';
+
+export default async function handler(_req, res) {
+    const jsonDirectory = path.join(process.cwd());
+    // @ts-ignore
+    const dummyVar = fs.readdirSync(`${jsonDirectory}/common`);
+    // @ts-ignore
+    const dummyLink = url.pathToFileURL(path.resolve(`${jsonDirectory}/common`));
+    const completedUpdate = await updateIndex();

```

Comment:
why do we need this API ? 
why cant this be done through script

Reply:
This has been created for instant rollback scenarios - build script won't be run again. Once the domain reassignment is done, we can run a github action that hits this endpoint
cc @bmbshlly